tests/admin_scripts/tests.py

```diff
@@ -2405,6 +2406,25 @@ def test_wrong_args(self):
         self.assertOutput(err, "usage:")
         self.assertOutput(err, "You must provide a project name.")
 
+    def test_output_directory_structure(self):
+
+        dirs = {
+            "testproject": self.test_dir,
+            "testproject1": os.path.join(self.test_dir, "otherdirectory"),
+        }
+        for name_project, dir in dirs.items():
+            with self.subTest(name_project=name_project, dir=dir):
+                args = ["startproject", name_project]
+                if not os.path.exists(dir):
+                    os.mkdir(dir)
+                    args.append(dir)
+                out, err = self.run_django_admin(args)
+
+                # We only check for a subset of the directory structure
+                self.assertOutput(out, name_project)
+                self.assertIn(f"{ELBOW_PREFIX}manage.py", out)
```

Comment:
```suggestion
                self.assertIn("|___ manage.py", out)
```
I wouldn't use the constants in tests